### ISSUE TITLE
Datepicker doesn't work with screen reader

### ISSUE THREAD

**Issue by 42tte** [SpareBank1/designsystem#498]
#### Which package is this issue related to?
ffe-datepicker-react

#### Describe your issue (screenshots welcome!)
The datepicker does not work with screen reader (VO/Safari) and needs thorough testing and overhauling. Exemple of bugs:
* role=button prevents screen reader from reading value of the input field (preselected date) and the fact that you are on an input field
* navigating the calendar dates nothing is read
* 

#### What is the expected behavior?
It should be usable with screen reader

#### What is the actual behavior?
It is not

#### Steps to reproduce
Use a screen reader on https://design.sparebank1.no/styleguidist/index.html#!/Datepicker

**Comment by pethel** [comment]
Kanske ta utgangspunkt i denne?

https://www.accessibility-developer-guide.com/examples/widgets/datepicker/

**Comment by kwltrs** [comment]
Fixed #977
